news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  "Call for paper for the PrestaShop Day Paris 2020"
+subtitle: "It's time to submit a talk to speak in front of the community"
+date:  2020-02-21 16:30:00
+authors: [AntoineThomas]
+icon: icon-board
+image: /assets/images/theme/meta-logo-build.png
+tags: [contribute, events, community]
+---
+
+
+Last year, a developer space was available at the PrestaShop Day Paris. This year, this space grows and will include an area for conferences and workshops dedicated to active contributors to the open source project.
+
+The event will take place in Paris, at "Palais Brongniart", June the 4th of 2020. More informations:  
+https://events.prestashop.com/prestashop-day-paris-2020/fr/
+
+## Thèmes
+
+Below is a list of themes for the call of papers that would be very appreciated this year.
+
+**Technique:**
+
+- FO or BO integration
+- Headless, PWA, API /. web-services
+- Information systems
```

Comment:
Quite vague

Reply:
yes. a lot of partners might buy a sponsored talk about integration in an existing information system, so I had to make this compromise.